packages/terra-date-picker/src/react-datepicker/index.jsx

```diff
@@ -768,7 +768,6 @@ class DatePicker extends React.Component {
       onClick: this.onInputClick,
       onFocus: this.handleFocus,
       onKeyDown: this.onInputKeyDown,
-      id: this.props.id,
```

Comment:
Why did we remove the id from the input being rendered?

Reply:
I will likely have to swap that over to being used by the input itself. Used here, it attempts to apply the same id to all inputs, of which does not make React happy.